src/display/editor/freetext.js

```diff
@@ -355,8 +355,26 @@ class FreeTextEditor extends AnnotationEditor {
     }
 
     this.disableEditMode();
-    this.#content = this.#extractText().trimEnd();
+    const savedText = this.#content;
+    const newText = (this.#content = this.#extractText().trimEnd());
+    if (savedText === newText) {
+      return;
```

Comment:
Previously we *unconditionally* called `this.#setEditorDimensions();` however that no longer happens, was that intended?

Reply:
this method is used to set the dimensions in percent of the editor so if its content  didn't change, the dimensions should be exactly the same as before so no need to set them again.
To answer to your question: yes it was.